Resources/Prototypes/Reagents/Consumable/Food/ingredients.yml

```diff
@@ -197,3 +197,29 @@
           type: Local
           messages: [ "capsaicin-effect-light-burn" ]
           probability: 0.2
+
+- type: reagent
+  id: CocoaPowder #Candy and chocolate
+  name: reagent-name-cacaopowder
+  group: Foods
+  desc: reagent-desc-cacaopowder
+  physicalDesc: reagent-physical-desc-coffee
+  flavor: caffeinated
```

Comment:
I'm not an expert on this stuff, but is cocoa caffeinated enough for this to make sense?

Reply:
Also, I can't find a `caffeinated` prototype anywhere, did you forget to add it?